lib/vcloud/launcher/vapp_orchestrator.rb

```diff
@@ -15,12 +15,15 @@ def self.provision(vapp_config)
         catalog_name = vapp_config[:catalog_name] || vapp_config[:catalog]
         template = Vcloud::Core::VappTemplate.get(template_name, catalog_name)
         template_id = template.id
-
         network_names = extract_vm_networks(vapp_config)
-        vapp = Vcloud::Core::Vapp.instantiate(name, network_names, template_id, vdc_name)
-        Vcloud::Launcher::VmOrchestrator.new(vapp.vms.first, vapp).customize(vapp_config[:vm]) if vapp_config[:vm]
+        if dry_run
+          Vcloud::Core.logger.info("Instantiating new vApp #{name} in vDC '#{vdc_name}'")
```

Comment:
Can you please include test output from a dry-run and non-dry-run run? I think this kind of message would be helpful in both cases.